src/databases/Mili/avtMiliFileFormat.C

```diff
@@ -810,6 +810,7 @@ avtMiliFileFormat::GetMesh(int timestep, int dom, const char *mesh)
 
         unsigned char *ghostZonePtr = ghostZones->GetPointer(0);
 
+        // TODO why is this loop duplicated?
```

Comment:
Where is the duplication?

Reply:
https://github.com/visit-dav/visit/blob/b73d24608586e7e6141dda14554b9a380bfb0f29/src/databases/Mili/avtMiliFileFormat.C#L800

and 

https://github.com/visit-dav/visit/blob/b73d24608586e7e6141dda14554b9a380bfb0f29/src/databases/Mili/avtMiliFileFormat.C#L814

I just want a note in there so that it isn't forgotten forever.

Reply:
I think it will be forgotten forever whether that note is there or not. I think the right answer is to delete the second loop

Reply:
ok done